desktop/src/gui/open_dialog.rs

```diff
@@ -155,26 +155,26 @@ impl OpenDialog {
                 );
                 ui.end_row();
 
-                ui.label(text(&self.locale, "socket-mode"));
-                ComboBox::from_id_source("open-file-advanced-options-socket-mode")
-                    .selected_text(match self.options.socket_mode {
-                        SocketMode::Unrestricted => text(&self.locale, "socket-mode-unrestricted"),
-                        SocketMode::Ask => text(&self.locale, "socket-mode-ask"),
-                        SocketMode::Deny => text(&self.locale, "socket-mode-deny"),
+                ui.label(text(&self.locale, "tcp-connections"));
+                ComboBox::from_id_source("open-file-advanced-options-tcp-connections")
+                    .selected_text(match self.options.tcp_connections {
+                        SocketMode::Unrestricted => text(&self.locale, "tcp-connections-allow"),
+                        SocketMode::Ask => text(&self.locale, "tcp-connections-ask"),
+                        SocketMode::Deny => text(&self.locale, "tcp-connections-deny"),
                     })
                     .show_ui(ui, |ui| {
                         ui.selectable_value(
-                            &mut self.options.socket_mode,
+                            &mut self.options.tcp_connections,
                             SocketMode::Unrestricted,
```

Comment:
SocketMode::Unrestricted should be renamed to SocketMode::Allow, if we want full parity with command line flag and desktop GUI.

Reply:
Good catch!